content/en/docs/concepts/services-networking/_index.md

```diff
@@ -1,63 +1,175 @@
+
+* [_Network plugins_](#network-plugins) - sets up IP addressing for pods and their containers, and allow pods to communicate
+even when the source pod and destination pod are running on different nodes. Different network plugins achieve this in
+different ways with examples including tunneling or IP routing.
+
+Processes with a pod, such as the processes within Pod 1, can communicate automatically. Kubernetes
+and the container runtime provide no special support as these processes all see a common local
+network within the container sandbox.
+
+You can also have connectivity between containers running on two or more different pods on the same node; for example
+Pod 7 communicating with Pod 1, with both Pods (and their containers) running on Node 1. The network plugin(s)
+that you deploy are responsible for the routes or other means to make sure that
+these packets arrive at the right destination.
+
+In the cross-node case, you have container communications between pods on nodes connected
+via the cluster network. In the example above, Pod 7 on Node 1 can talk to Pod 21 on Node 2.
+
+{{< note >}}
+The network model permits all pods to talk to all other pods on the cluster. However, you might implement policies in your cluster to limit what pods can talk to other pods.
+{{< /note >}}
+
+The network model describes how pods and their associated pod IPs can integrate with the larger network to support
+container networking.
+
+[comment]: <> (All diagrams.net figures are available at: https://drive.google.com/drive/folders/1MPOeuJ3wTzptutZX_6GKpLK8ljnojKE8?usp=sharing)
+
+[comment]: <> (good talk on K8 network models at https://www.cncf.io/wp-content/uploads/2020/08/CNCF_Webinar_-Kubernetes_network_models.pdf)
+
+Kubernetes IP addresses exist at the Pod scope. For example, on Linux, containers
+within a Pod share their network namespaces - including their IP address, and any
+network address from a lower layer, such as a MAC address.
+This means that containers within a Pod can all reach each other's ports on
+`localhost`. This also means that containers within a Pod must coordinate port
+usage (the same way that different processes on a physical server need to coordinate
+port use. This model, as used in Kubernetes, is called the _IP-per-pod_ model.
 
 How this is implemented is a detail of the particular container runtime in use.
 
-It is possible to request ports on the `Node` itself which forward to your `Pod`
-(called host ports), but this is a very niche operation. How that forwarding is
-implemented is also a detail of the container runtime. The `Pod` itself is
-blind to the existence or non-existence of host ports.
+It is possible to request and configure ports on the node itself (named _host ports_),
+that forward to a port on your Pod; however, this is a very niche operation.
+How that forwarding is implemented is also a detail of the container runtime.
+The Pod itself is not aware of the existence or non-existence of host ports.
```

Comment:
>  It would be great to have small sections explicitly comparing Kubernetes networking to (a) traditional host networking, (b) Docker networking, (c) VM (eg OpenStack) networking. (I'm not sure if we need to talk about non-Kubernetes cloud networking since that generally tries to look like traditional host networking?)

I agree, but again I'm looking to find the minimal diff from what we have to what we're willing to merge. The perfect is the enemy of the published.